### PR TITLE
[IMP] event_planned_by_sale_line: New report with counters and invoic…

### PR DESCRIPTION
**Pull request by alfredoavanzosc**
…es for event registrations.
Nuevo informe de contadores de registros con facturas.

### CODE REVIEW

File: event_planned_by_sale_line/i18n/es.po [hunk 1]
```diff
@@ -38,40 +86,105 @@ msgid "End hour"
...
+
+#. module: event_planned_by_sale_line
+#: field:account.invoice.line,invoice_state:0
+msgid "Invoice state"
+msgstr "Estado factura"
+
+#. module: event_planned_by_sale_line
+#: help:account.invoice.line,invoice_date_invoice:0
+msgid "Keep empty to use the current date"
+msgstr "Keep empty to use the current date"
```
Comment (Daniel-CA) [inline event_planned_by_sale_line/i18n/es.po]: Esta linea no está traducida

File: event_planned_by_sale_line/i18n/es.po [hunk 2]
```diff
@@ -83,6 +196,27 @@ msgid "Punctual service"
...
+#. module: event_planned_by_sale_line
+#: model:ir.actions.act_window,name:event_planned_by_sale_line.action_event_registration_invoice_report_view
+#: model:ir.ui.menu,name:event_planned_by_sale_line.menu_action_event_registration_invoice_report
+msgid "Report registrations with invoices counters"
+msgstr "Informe contador registros con facturas"
+
+#. module: event_planned_by_sale_line
+#: model:ir.model,name:event_planned_by_sale_line.model_event_registration_invoice_report
+msgid "Report with counters and invoices for event registrations"
+msgstr "Inform contador registros"
```
Comment (Daniel-CA) [inline event_planned_by_sale_line/i18n/es.po]: Informe

**Comment by Daniel-CA** [comment]
:+1:

**Review by mikelarre — APPROVED** [review]
:+1:

**Pull request merged**